src/uct/ib/rc/verbs/rc_verbs_iface.c

```diff
@@ -235,8 +462,11 @@ static ucs_status_t uct_rc_verbs_iface_tag_init(uct_rc_verbs_iface_t *iface,
         srq_init_attr.pd                  = md->pd;
         srq_init_attr.cq                  = iface->super.super.recv_cq;
         srq_init_attr.tm_cap.max_num_tags = iface->tm.tag_available;
-        srq_init_attr.tm_cap.max_tm_ops   = ucs_min(2*iface->tm.tag_available,
-                                            IBV_DEVICE_TM_CAPS(&md->dev, max_tag_ops));
+
+        /* 2 ops for each tag (ADD + DEL) and extra 8 tags for SYNC ops.
+         * Do not rely on max_tag_ops device capability, so that XRQ creation
+         * would fail if such # of ops is not supported (which is not expected). */
+        srq_init_attr.tm_cap.max_tm_ops   = 2 * iface->tm.tag_available + 8;
```

Comment:
where is the number '8' coming from? could it be calculated / #defined ? does it depend on IBV_DEVICE_MAX_UNEXP_COUNT?